lib/types.d.ts

```diff
@@ -215,10 +215,15 @@ type NonTransformableTypes =
 type NullishDeep<T> = T extends NonTransformableTypes ? T : T extends object ? NullishObjectDeep<T> : unknown;
 
 /**
-Same as `NullishDeep`, but accepts only `object`s as inputs. Internal helper for `NullishDeep`.
-*/
+ * Same as `NullishDeep`, but accepts only `object`s as inputs. Internal helper for `NullishDeep`.
+ */
 type NullishObjectDeep<ObjectType extends object> = {
     [KeyType in keyof ObjectType]?: NullishDeep<ObjectType[KeyType]> | null;
 };
 
-export {CollectionKey, CollectionKeyBase, CustomTypeOptions, DeepRecord, Key, KeyValueMapping, OnyxCollection, OnyxEntry, OnyxKey, Selector, NullishDeep};
+/**
+ * Represents withOnyx's internal state, containing the Onyx props and a `loading` flag.
+ */
+type WithOnyxInstanceState<TOnyxProps> = (TOnyxProps & {loading: boolean}) | undefined;
```

Comment:
Just a question, is loading always a boolean or could be undefined as well?

Reply:
Looks like it's being defined since the beginning [here](https://github.com/Expensify/react-native-onyx/blob/main/lib/withOnyx.js#L91). I didn't find evidence of it being set to undefined